subsys/bluetooth/audio/has.c

```diff
@@ -474,6 +482,8 @@ static void preset_changed_prepare(struct net_buf_simple *buf, uint8_t change_id
 	preset_changed = net_buf_simple_add(buf, sizeof(*preset_changed));
 	preset_changed->change_id = change_id;
 	preset_changed->is_last = is_last;
+
+	/** set pending */
```

Comment:
Is something missing here?

Reply:
Sorry, that was just a note I had written for myself that no longer applies :D